helm/charts/hpe-csi-driver/templates/hpe-csi-controller.yaml

```diff
@@ -26,17 +26,17 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: csi-provisioner
-          {{- if semverCompare "^1.12.0" .Capabilities.KubeVersion.GitVersion }}
+          {{- if semverCompare "~1.12.0" .Capabilities.KubeVersion.GitVersion }}
           image: {{ .Values.csiProvisionerImage}}: {{- .Values.csiProvisionerTagv0}}
           {{- else }}
           image: {{ .Values.csiProvisionerImage}}: {{- .Values.csiProvisionerTagv1}}
           {{- end }}
           args:
             - "--csi-address=$(ADDRESS)"
             - "--v=5"
-            {{- if semverCompare ">= 1.13.0" .Capabilities.KubeVersion.GitVersion }}}
+            {{- if semverCompare ">= 1.13.0" .Capabilities.KubeVersion.GitVersion }}
             - "--timeout=30s"
-            {{- end }}}
+            {{- end }}
```

Comment:
Thanks :)

Reply:
NP George :)..I usually run "helm lint" on my Mac that caught it..